### PR TITLE
feature: Add packaging game

### PR DESCRIPTION
**Pull request by alexfauquette**
add `/packaging` pagewith basic table for now

### CODE REVIEW

**Comment by teolemon** [comment]
10:59:26 PM: src/pages/packaging/index.tsx
10:59:26 PM:   Line 62:16:  'setRows' is assigned a value but never used  @typescript-eslint/no-unused-vars

File: src/pages/packaging/index.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  const [rows, setRows] = React.useState([{}]);
+  return (
+    <TableContainer component={Paper}>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>Nb per unit</TableCell>
+
+            <TableCell>Shpae</TableCell>
```
Comment (teolemon) [inline src/pages/packaging/index.tsx:70]:
```suggestion
            <TableCell>Shape</TableCell>
```

File: src/pages/packaging/index.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    </TableRow>
+  );
+};
+const Page = (props) => {
+  const lang = getLang();
+  const packagingMaterials = useOptions("packaging_materials", lang);
+  const packagingShapes = useOptions("packaging_shapes", lang);
+  const packagingRecycling = useOptions("packaging_recycling", lang);
+
+  const [rows, setRows] = React.useState([{}]);
```
Comment (teolemon) [inline src/pages/packaging/index.tsx:62]: defined but not used

**Comment by alexfauquette** [comment]
Can be tested here:
https://deploy-preview-363--warm-cannoli-79bbb2.netlify.app/packaging

For now, it does not send update to the serve. only log in the console the data collected

**Pull request merged**